spec/features/add_team_to_case_spec.rb

```diff
@@ -46,6 +47,10 @@
     expect(page).to have_selector("a", text: "Select the permission level the team should have")
     expect(page).to have_selector("a", text: "Select whether you want to include a message")
 
+    # Check deleted teams are not listed
+    expect(page).to have_css("#team option[value=\"#{team.id}\"]")
+    expect(page).not_to have_css("#team option[value=\"#{deleted_team.id}\"]")
```

Comment:
You could use the Capybara method here to check the values of the select?

```suggestion
    # Check deleted teams are not listed
    expect(page).not_to have_select("Select other team name", with_options: [deleted_team.name])
    expect(page).to have_select("Select other team name", with_options: [team.name])
```